Introduction.Rmd

```diff
@@ -137,64 +159,97 @@ Make sure you have a recent version of RStudio. You can check you have the right
 
 ```{r, eval = FALSE}
 install.packages("rstudioapi")
+# TODO: update this version number?
 rstudioapi::isAvailable("0.99.149")
 ```
 
-If not, you may need to install the preview version from <http://www.rstudio.com/products/rstudio/download/preview/>. This gives you access to the latest and greatest features, and only slightly increases your chances of finding a bug. 
+If not, you may need to install the preview version from <https://www.rstudio.com/products/rstudio/download/preview/>. This gives you access to the latest and greatest features, and only slightly increases your chances of finding a bug. 
 
-Use the following code to install the development versions of devtools and usethis. This is particularly important during the development of the book.
+The following code installs the development versions of devtools and usethis, which may be important during the revision of the book.
 
 ```{r, eval = FALSE}
 devtools::install_github("r-lib/devtools")
 devtools::install_github("r-lib/usethis")
 ```
 
-You'll also need a C compiler and a few other command line tools. If you're on Windows or Mac and you don't already have them, RStudio will install them for you. Otherwise:
+You'll also need a C compiler and a few other command line tools.
+
+If you're on Windows or Mac and you don't already have them, RStudio will install them for you. TO CHECK: is this really true? I've added instructions, for now.
```

Comment:
Does RStudio really take care of this? Regardless, perhaps it is nice to provide instructions.

Reply:
Most of what I added came from WTF, with some roots in Happy Git and STAT 545.

Reply:
It's definitely true.

Reply:
Yeah but RStudio does some sub-optimal things I believe, there was an issue at one point in pkgbuild about it. I think it is definitely worth investigating. See https://github.com/r-lib/pkgbuild/issues/65 and https://github.com/rstudio/rstudio/issues/3563